cmd/common/utils.go

```diff
@@ -85,6 +86,13 @@ func CheckSourcePath(srcpath string) {
 	if !fi.IsDir() {
 		log.Fatalf("The given source path %s is a file. Expected a directory. Exiting.", srcpath)
 	}
+	pwd, err := os.Getwd()
```

Comment:
Why are we concerned about the current working directory at all? Shouldn't we be just comparing between the source and output directories?

Reply:
to check if the source is a parent of the current working directory.

Reply:
```
we be just comparing between the source and output directories?
```
we are doing that too